spec/models/comment_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  let(:user) { create(:user) }
+  let(:post) { create(:post, author: user) }
+
+  subject(:comment) { described_class.new(text: 'Test comment', author: user, post: post) }
+
+  describe 'associations' do
+    it { should belong_to(:author).class_name('User').with_foreign_key(:author_id) }
+    it { should belong_to(:post) }
+  end
```

Comment:
- [ ] As required for the project any custom methods in the models to have a corresponding unit test, Kindly ensure that you add unit test(s) to `#update_comments_counter` method in the `Comments` class.